user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+                Name = "JobTitle",
+                Tooltip = "Job title of the person"
+            };
+
+            // Put all the fieldDescriptors in a sectionDefinition
+            var sectionDefinition = new CustomSectionDefinition()
+            {
+                Name = "PeopleSectionDefinition"
+            };
+
+            sectionDefinition.AddOrReplaceFieldDescriptor(firstNameFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(lastNameFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(phoneNumberFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(jobTitleFieldDescriptor);
+
+            sectionDefinition = _domHelper.SectionDefinitions.Create(sectionDefinition) as CustomSectionDefinition;
+
+            // Create the domDefinition
+            var personDomDefinition = new DomDefinition()
+            {
+                Name = "PersonDomDefinition",
+                SectionDefinitionLinks = new List<SectionDefinitionLink>()
+                {
+                    new SectionDefinitionLink(sectionDefinition?.GetID())
+                }
+            };
+
+            personDomDefinition = _domHelper.DomDefinitions.Create(personDomDefinition);
+
+            var sectionDefinitionId = sectionDefinition?.ID;
+            var firstNameFieldDescriptorId = firstNameFieldDescriptor.ID;
+            var lastNameFieldDescriptorId = lastNameFieldDescriptor.ID;
+            var phoneNumberFieldDescriptorId = phoneNumberFieldDescriptor.ID;
+            var jobTitleFieldDescriptorId = jobTitleFieldDescriptor.ID;
+            var domDefinitionId = personDomDefinition.ID;
+
+            // Create a list of FieldValues for the general Section
+            var fieldValues = new List<FieldValue>()
+            {
+                new FieldValue(firstNameFieldDescriptorId, new ValueWrapper<string>("John")),
+                new FieldValue(lastNameFieldDescriptorId, new ValueWrapper<string>("Doe")),
+                new FieldValue(phoneNumberFieldDescriptorId, new ValueWrapper<string>("0423482635")),
+                new FieldValue(jobTitleFieldDescriptorId, new ValueWrapper<int>(0)),
+            };
+
+            // Add them to a Section linked to the ID of the GeneralSectionDefinition
+            var generalSection = new Section()
+            {
+                SectionDefinitionID = sectionDefinitionId
+            };
+
+            foreach (var fieldValue in fieldValues)
+            {
+                generalSection.AddOrReplaceFieldValue(fieldValue);
+            }
+
+            // Create of the actual DomInstance
+            var domInstance = new DomInstance()
+            {
+                DomDefinitionId = domDefinitionId,
+                Sections = { generalSection }
+            };
+            domInstance = _domHelper.DomInstances.Create(domInstance);
+        }
+    }
+}
+```
+
+To visualize the setup, create a low code app. On that low code app:
```

Comment:
Not entirely sure whether we should explain this in detail here. In theory, how you can configure a DOM form should be documented somewhere else. (Not sure if that is already the case?) Either way, I would suggest specifying that we are talking about GQI queries and maybe that the items should be dragged on the 'canvas' or something like that. (Maybe check the docs about app editing and see how it worded there)